README.md

```diff
@@ -4,7 +4,9 @@ This may be useful either:
 - for manual testing/experimentation. Different cloud functions can be deployed from the same source code, so you can deploy to a test function without affecting any of the other resources.
 
 ### One-time setup
-Install Cloud SDK ([Quickstart](https://cloud.google.com/sdk/docs/quickstart))
+Install Cloud SDK ([Quickstart](https://cloud.google.com/sdk/docs/quickstart))  
+Install Terraform ([Getting started](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/gcp-get-started))  
```

Comment:
This is nested under the "Deploying functions manually" header that I added. I would either create a separate "One-time setup" section under "Deploying your own instance with terraform" or hoist this to the top level